docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -1409,6 +1415,160 @@ spec:
 {% endnavtab %}
 {% endnavtabs %}
 
+#### Node Selectors & Tolerations
+
+Kubernetes provides many options to control the scheduling of pods
+onto nodes in the cluster. The most direct of which is a [node selector](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector).
+
+However, Kubernetes will avoid scheduling pods on any tainted nodes.
+To allow certain pods to schedule on such nodes, Kubernetes allows
+you to specify tolerations for those taints. See [the Kubernetes documentation
```

Comment:
Do hyper links work over multiple lines?

Reply:
When I rendered this using `jekyll serve` the hyperlink displayed and functioned correctly.

Reply:
ah great. I was concerned because github doesn't seem to recognize this multi-line `[...]` (see the color is slightly different from other hyper links).